### DISCUSSION TITLE
Light and Dark Mode Support

### DISCUSSION THREAD

**Discussion by imparth7** [VirgoLabs/doclab#8] [Q&A] [answered]
Hello everyone,

I hope this message finds you well! I wanted to inquire about the possibility of incorporating light and dark mode support in the DocLab project.

As documentation creation can often span extended periods, having the option to switch between light and dark themes would greatly enhance user experience and accessibility.

Are there any plans for this feature, or could it be considered for a future release? I believe it would benefit many users.

Thank you for your hard work on this project!

**Comment by ParthVirgoz** [comment, marked as answer]
Great question! I’m happy to confirm that DocLab does support both light and dark modes.

If you have any more questions or suggestions, feel free to share!
